tests/token_provision.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library     SSHLibrary    timeout=90 seconds
+Library     Process
+Library     OperatingSystem
+Library     String
+Library     RequestsLibrary
+Library     Collections
+
+Suite Setup       Run Keyword    Open Server
+
+Resource    ../variables.robot
+Resource    ../keywords.robot
+
+*** Test Cases ***
+
+TOP001.001 Token provisioning
+    [Documentation]    Checks that API command admin/token_provision works
+    ...                correctly.
+    [Teardown]    Run Keyword If Test Failed    Set Suite Variable    ${prev_test_status}    FAIL
+    Run    python3 provisioning/cc_cbor.py provisioning/chain.pem provisioning/tmp/fobnail.cbor
+    Run    coap-client -t application/cbor -m post -f provisioning/tmp/fobnail.cbor coap://169.254.0.1/api/v1/admin/token_provision > provisioning/tmp/fobnail.csr
+    SSHLibrary.Read Until    Certificate chain loaded
+    SSHLibrary.Read Until    Generating new
```

Comment:
What happens if server doesn't respond with these messages? Wouldn't it cause the test to hang forever if Fobnail responded with anything else?
Anyway, the test should not rely on data printed from server, but on server response - [Fobnail API docs](https://fobnail.3mdeb.com/fobnail-api/#admintoken_provision) describe how the response should look like, in that case server should respond with **2.01** code and this is what the test should check.

Reply:
It will fail after timeout. I never got a response in 2.01 format, so it's hard to lean on. If there are any conditions to be met to receive this result, please inform me. I started the server with `--verbose-mode` and `-v 9` and it didn't work as expected.

Reply:
applied